### PR TITLE
Update Liquibase version to 3.8.7

### PR DESCRIPTION
**Pull request by DanielFran**
See:
- https://github.com/liquibase/liquibase-hibernate/issues/222
- https://github.com/liquibase/liquibase/pull/927
---

-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/server/templates/build.gradle.ejs [hunk 1]
```diff
@@ -243,8 +243,8 @@ configurations {
     all {
         resolutionStrategy {
             // Inherited version from Spring Boot can't be used because of regressions:
-            // https://liquibase.jira.com/browse/CORE-3536 and https://liquibase.jira.com/browse/CORE-3522
```
Comment (murdos) [inline generators/server/templates/build.gradle.ejs]: These issues (https://liquibase.jira.com/browse/CORE-3536 and https://liquibase.jira.com/browse/CORE-3522) are still open. Thus are you sure it's safe to upgrade?

Comment (DanielFran) [reply generators/server/templates/build.gradle.ejs]: This seems fixed in the changelog of 3.8.1+, but I am not sure... Even the issue with Java 11 should be fixed.

**Comment by DanielFran** [comment]
@pascalgrimaud @murdos @Tcharl Any objection to merge it?

**Comment by pascalgrimaud** [comment]
No objection but it will be conflict with https://github.com/jhipster/generator-jhipster/pull/11405

**Comment by murdos** [comment]
No objection. Regarding #11405 don't worry, I'll resolve the conflicts.

**Pull request merged**